src/sources/util/tcp.rs

```diff
@@ -121,15 +135,8 @@ pub trait TcpSource: Clone + Send + 'static {
 
                     let source = self.clone();
                     span.in_scope(|| {
-                        accept_socket(
-                            span.clone(),
-                            socket,
-                            source,
-                            tripwire,
-                            host,
-                            out.clone(),
-                            tls.clone(),
-                        )
+                        debug!("accepted a new connection");
```

Comment:
we could include the remote addr here.

Reply:
Good idea. The old code didn't report the remote address neither, but it's still a good idea.

Reply:
We might be able to append it to the span too via https://docs.rs/tracing/0.1.13/tracing/struct.Span.html#method.record